resources/qml/Dialogs/OpenFilesIncludingProjectsDialog.qml

```diff
@@ -43,69 +35,31 @@ UM.Dialog
         }
     }
 
-    Column
+    onAccepted: loadModelFiles(base.fileUrls)
+
+    UM.Label
     {
-        anchors.fill: parent
-        anchors.leftMargin: 20 * screenScaleFactor
-        anchors.rightMargin: 20 * screenScaleFactor
-        anchors.bottomMargin: 20 * screenScaleFactor
+        text: catalog.i18nc("@text:window", "We have found one or more project file(s) within the files you have selected. You can open only one project file at a time. We suggest to only import models from those files. Would you like to proceed?")
         anchors.left: parent.left
         anchors.right: parent.right
-        spacing: 10 * screenScaleFactor
-
-        Label
-        {
-            text: catalog.i18nc("@text:window", "We have found one or more project file(s) within the files you have selected. You can open only one project file at a time. We suggest to only import models from those files. Would you like to proceed?")
-            anchors.left: parent.left
-            anchors.right: parent.right
-            font: UM.Theme.getFont("default")
-            wrapMode: Text.WordWrap
-        }
-
-        Item // Spacer
-        {
-            height: base.spacerHeight
-            width: height
-        }
-
-        UM.I18nCatalog
-        {
-            id: catalog
-            name: "cura"
-        }
-
-        ButtonGroup
-        {
-            buttons: [cancelButton, importAllAsModelsButton]
-            checkedButton: importAllAsModelsButton
-        }
+        font: UM.Theme.getFont("default")
+        wrapMode: Text.WordWrap
```

Comment:
```suggestion
```
These values are the default for UM.Label. There is no need to re set them (DRY)